src/module/compcon.ts

```diff
@@ -1,3 +1,4 @@
+import { AUDIO_FILE_EXTENSIONS } from "@league-of-foundry-developers/foundry-vtt-types/src/foundry/common/constants.mjs";
```

Comment:
What's going on here?

Reply:
That's an interesting auto-include from vscode - i'll remove it in a commit now.